api/specs.proto

```diff
@@ -293,6 +293,23 @@ message ContainerSpec {
 	// task will exit and a new task will be rescheduled elsewhere. A container
 	// is considered unhealthy after `Retries` number of consecutive failures.
 	HealthConfig healthcheck = 16;
+
+	enum Isolation {
+		option (gogoproto.goproto_enum_prefix) = false;
+
+		// Default uses whatever default value from the container runtime
+		Default = 0 [(gogoproto.enumvalue_customname) = "ContainerIsolationDefault"];
```

Comment:
Please use all caps for enums. See the `TaskState` for an example of enum style. Remember that these are scoped to the package from protobufs perspective, so they should be prefixed:

`ISOLATION_DEFAULT`
`ISOLATION_PROCESS`

The Go names you have in place are perfect.